apps/backend/src/app/api/v1/users/crud.tsx

```diff
@@ -638,6 +642,28 @@ export const usersCrudHandlers = createLazyProxy(() => createCrudHandlers(usersC
         }
       }
 
+
+      // Hacky passkey auth method crud, should be replaced by authHandler endpoints in the future
+      if (data.passkey_auth_enabled !== undefined) {
+        if (data.passkey_auth_enabled) {
+          // Case: passkey_auth_enabled is set to true. This should only happen after a user added a passkey and is a no-op since passkey_auth_enabled is true iff there is a passkey auth method.
+          // Here to update the ui for the settings page.
+          // The passkey auth method is created in the registerPasskey endpoint!
```

Comment:
won't this also happen if the user makes a custom request to `/user/me` and sets `passkey_auth_enabled: true`? In that case, if there is no passkey registered we should throw an error to let the user know that they can't do that